pagopa/io-app
# chore: [IOBP-195] Dynamic padding to bottom sheet into PDNDAcceptance screen

## Short description
This PR fixes the bottom sheet bottom padding into PDNDAcceptance screen IDPay, setting it equal to the default padding 130 + the calculated height of the footer.

## List of changes proposed in this pull request
- `PDNDPrerequisitesScreen`: added a new state to calculate dynamically the height of the footer and adding that value to the padding bottom.

## How to test
Start the onboarding process of any IDPay initiative and open the bottom sheet into the `IDPAY_ONBOARDING_PDNDACCEPTANCE` screen, it should shows correctly

## Preview

|Before|After|
|-|-|
|<img width="350" alt="before" src="https://github.com/pagopa/io-app/assets/34343582/72f5cde4-1a04-4320-ad41-d94324e5d772"/>|<video src="https://github.com/pagopa/io-app/assets/34343582/bd2ae41d-a451-4443-bd2f-bcbe759b415a" />|
